src/sections/FAQ/questions.tsx

```diff
@@ -0,0 +1,53 @@
+import Typography from '@/components/Typography';
+import Link from 'next/link';
+
+interface Question {
+  question: string;
+  answer: string | React.ReactNode;
+}
+
+export const QUESTIONS: Question[] = [
+  {
+    question: 'How can I register for DiamondHacks?',
+    answer: (
+      <Typography variant="body/medium">
```

Comment:
Maybe it would be better to use a fragment <>...</>. This way if the typography needs to be changed you wouldn't have to update the questions data as well

Maybe for consistency, you could make all of the answers use JSX. This way if formatting like links needs to be added to an answer in the future, it'll be easier to add